src/tabwidget.cpp

```diff
@@ -276,8 +278,17 @@ bool TabWidget::eventFilter(QObject *obj, QEvent *event)
         int index = tabBar()->tabAt(e->pos());
         if (index == -1)
         {
-            TerminalConfig defaultConfig;
-            addNewTab(defaultConfig);
+            if (Properties::Instance()->terminalsPreset == 3)
+                preset4Terminals();
```

Comment:
Could you add curly brackets for all if/else blocks? I always do that after Apple's "goto fail;" bug revealed.

Reply:
Sure.